backend/profiles/migrations/0003_auto_20190402_0026.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.1.7 on 2019-04-02 00:26
```

Comment:
Same thing here - go ahead and pull in staging; I think this migration will be extraneous.